spec/dfe/analytics/requests_spec.rb

```diff
@@ -5,13 +5,18 @@
     controller = Class.new(ApplicationController) do
       include DfE::Analytics::Requests
     end
+    public_api_controller = Class.new(PublicApiController) do
+      include DfE::Analytics::Requests
+    end
 
     stub_const('TestController', controller)
+    stub_const('TestPublicApiController', public_api_controller)
```

Comment:
Is this name pulling in a bit too much context from Publish? the important thing is that there is no user, so how about something like `TestUnauthenticatedController`?

Reply:
Sure. I'll rename.